server/app/views/applicant/IneligibleTemplate.html

```diff
@@ -5,6 +5,9 @@
   <body>
     <div th:replace="~{applicant/NavigationFragment :: pageHeader}"></div>
     <div role="main" class="margin-2">
+      <div
+        th:replace="~{components/ToastFragment :: devOrStagingToast(${isDevOrStaging})}"
```

Comment:
praise:  I like that you're just inserting the `devOrStagingToast` fragment directly rather than making a toast fragment for this page, since there's just one toast.